shops/src/main/resources/plugin.yml

```diff
@@ -4,4 +4,4 @@ name: Shops
 load: POSTWORLD
 version: '1.0'
 depend: [Core]
-api-version: 1.20
\ No newline at end of file
+api-version: '1.20'
```

Comment:
You might as well update the others as well, maybe just do a find and replace for 

api-version: 1.19
with
api-version '1.20'